docs/development/xenial_support.rst

```diff
@@ -13,22 +13,42 @@ based on Xenial.
 Running the Xenial dev env
 --------------------------
 
+The Xenial staging environment uses libvirt as a VM provider. If you haven't 
+already done so, you should follow the instructions here: :ref:`libvirt_provider`.
+
+Once the libvirt Vagrant provider is available, you'll need a libvirt-format Xenial
+base image. To set one up, run the following commands.
```

Comment:
Again, virtualbox _should_ be supported, as well. I haven't actually been testing that, so I'd appreciate confirmation from someone on a Mac if possible.

Reply:
Gonna take it on faith and run through the flow on a Mac when I get a chance :)